examples/Cargo.toml

```
@@ -20,5 +20,5 @@ path = "src/ext_authz/authorization_server_pb_only.rs"
 
 [dependencies]
 envoy-types = { path = "../envoy-types" }
-tokio = { version = "1.29.1", features = ["rt-multi-thread"] }
-tonic = "0.9.2"
+tokio = { version = "1.35.1", features = ["rt-multi-thread"] }
```

```suggestion
tokio = { version = "1.35.1", features = ["macros", "rt-multi-thread"] }
```
We need this feature to run our examples.